### DISCUSSION TITLE
Is it possible to login the Cluster with oc CLI from a different machine where CRC is installed?

### DISCUSSION THREAD

**Discussion by eduizquierdo** [crc-org/crc#2952] [Q&A]
I installed CRC on WIndows 11. I can "oc login" from my Windows system without problems.

Now I need to login from Linux to run a script only available for Linux, but I'm having the error: `dial tcp 127.0.0.1:6443: connect: connection refused - verify you have provided the correct host and port and that the server is currently running.`
 
I'm trying from Windows Linux Services WSL2 (Ubuntu). This is what I did:
* copied the connection settings (.crc/machines/crc/kubeconfig) to my Linux home directory (~/.kubeconfig)
* installed oc CLI
* copied the relevant host names added by CRC install from WIndows (System32\drivers\etc\hosts) to Linux (etc/hosts)

Is there something that I'm missing? 
Is it possible to login the cluster with 'oc login' from a different OS where CRC was installed?

Thanks in advance.

**Comment by cfergeau** [comment]
How do you usually access the host from WSL2? I'd try replacing `127.0.0.1` with the IP of the host if usually a different one is used. But not fully sure this is going to be accessible without more tweaks.

**Comment by gbraad** [comment]
WSL2 can not communicate with VMs directly. You need to re-enable this on each restart: https://github.com/code-ready/crc/issues/374#issuecomment-803580645 Since we now use usermode networking I think you only need to enable forwarding for the WSL switch.
